t/cli.c

```diff
@@ -358,6 +359,7 @@ static void usage(const char *cmd)
            "  -E esni-file         file that stores ESNI data generated by picotls-esni\n"
            "  -e                   when resuming a session, send first 8,192 bytes of input\n"
            "                       as early data\n"
+           "  -r                   Load raw public keys for raw certificates\n"
```

Comment:
Regarding your concern, I think it would be a good idea to define `-r` as:
* an option to use raw public keys for validating the server, and
* the optarg of as the identity that the server is expected to provide. When `-r` is used on the server-side, we will simply ignore the optarg.

The benefit is that then `-c` can always be an option to set endpoint's identity (regardless of that identity being X.509 or raw public keys).

WDYT?

Assuming that we adopt the design, the help text would be something like:

```suggestion
           "  -r                   Load raw public keys for raw certificates\n"
           "  -r public-key-file   use raw public keys (RFC 7250). When set and running as a\n"
           "                       client, the argument specifies the public keys that the\n"
           "                       server is expected to use. When running as a server, the\n"
           "                       argument is ignored.\n"
```